src/widgets/StepperInput.js

```diff
@@ -15,6 +15,9 @@ import { AddIcon, MinusIcon } from './icons';
  * The buttons can also be held, increase by one every 50ms. The increment amount
  * then grows exponentially every 500ms, reseting to one after releasing.
  *
+ * Component uses state to update itself, and a ref for storing the current value. This
+ * is a bit of a hack, but is much more responsive to the rapid updates of the input.
```

Comment:
maybe just me, but I don't see it as that hacky. For a component like this it is sensible to do the rendering based on local state. And my understanding of the `useRef` is that it is perfect for this sort of thing.

Reply:
I guess more of a creative use of tools? 🤣